### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by dustindall** [Azure/rAzureBatch#29]
Do you plan on adding documentation to this package and the available functions?

**Comment by brnleehng** [comment]
Yes, most definitely. I've been planning on adding some documentation for people who want to build on top of rAzureBatch and using storage api calls. It should be coming in a couple of days

**Comment by kwiscion** [comment]
Any update on this? Documentation would be really helpful.
